src/app-e-heat-input-from-gas-workspace/app-e-heat-input-from-gas-workspace.service.ts

```diff
@@ -93,14 +97,15 @@ export class AppEHeatInputFromGasWorkspaceService {
       updateDate: timestamp,
     });
 
-    await this.repository.save(entity);
+    await repository.save(entity);
 
     entity = await this.repository.getAppEHeatInputFromGasById(entity.id);
```

Comment:
This should be updated to use the transactional repository:
```suggestion
    entity = await repository.getAppEHeatInputFromGasById(entity.id);
```

Reply:
updated.